static/app/views/app/index.spec.tsx

```diff
@@ -47,13 +65,14 @@ describe('App', function () {
     jest.resetAllMocks();
   });
 
-  it('renders', function () {
+  it('renders', async function () {
     render(
       <App {...routerProps}>
         <div>placeholder content</div>
       </App>
     );
 
+    await waitFor(() => OrganizationsStore.getAll().length === 1);
```

Comment:
Need this to avoid `act` errors since the `OrganizationContext` is subscribed to the organizations list store.